### ISSUE TITLE
Steam native to flatpak cloud sync bug

### ISSUE THREAD

**Issue by telometto** [ValveSoftware/steam-for-linux#7662]
Proton experimental:
Doesn't seem to be an issue with Proton itself, but rather migrating from the native Linux client to the flatpak version. After I migrated to the flatpak version, my saved games seem to be gone. Not much help in Steam help either, as they seem to answer me with copypastas.

Therefore, peeps, **BACK UP YOUR SAVED GAMES TO A USB-DRIVE!** Better safe than sorry.

**Comment by kisak-valve** [comment]
Hello @telometto, Steam's cloud sync in handled by the Steam client, not Proton itself, so I've transferred this issue report to the steam-for-linux issue tracker.

**Comment by telometto** [comment]
> https://store.steampowered.com/account/remotestorage
> 
> Your save games should be there.

Yeah, I checked. Unfortunately, they aren't. What's odd is the fact that I have formatted the PC and reinstalled twice but it was on the second time that the games disappeared.

> Hello @telometto, Steam's cloud sync in handled by the Steam client, not Proton itself, so I've transferred this issue report to the steam-for-linux issue tracker.

Alright, thank you.
